transforms/ember-object/__testfixtures__/decorators.output.js

```diff
@@ -67,11 +67,12 @@ class Foo extends EmberObject {
   Computed fullname
   */
   @computed("firstName", "lastName")
-  get fullName(key) {
+  @readOnly
```

Comment:
It looks like `@readOnly` and `@volatile` aren't ever imported, you should be able to import them from `"@ember-decorators/object";`

Reply:
Ahh! Nice catch. Thanks!

Reply:
Changes added to import `readOnly` and `volatile` from `"@ember-decorators/object"`. Don't know why the jscodeshift adds an empty line in between though